moonstreamapi/moonstreamapi/routes/queries.py

```diff
@@ -117,6 +121,7 @@ async def create_query_handler(
             title=f"Query:{query_name}",
             tags=["type:query"],
             content=query_applied.query,
+            timeout=10,

```

Comment:
We have variable in settings.py `BUGOUT_REQUEST_TIMEOUT_SECONDS` what if we will use something like `BUGOUT_REQUEST_TIMEOUT_SECONDS * 2` ? At lease we will reduce hardcoded timeouts, or we can create addtional variable